### DISCUSSION TITLE
Why is crepe_tiny and rmvpe_onnx the only F0 Detection thats available in the onnxdirectML-cuda version?

### DISCUSSION THREAD

**Discussion by AverageMinecraftEnjoyer** [w-okada/voice-changer#1225] [Q&A]
the 2 listed F0 Detections are the only ones available for me at least, why?
![Screenshot 2024-06-03 224514](https://github.com/w-okada/voice-changer/assets/96318403/8ee2b4e2-9e49-4cd5-8041-ec39ef250309)

**Comment by AnnoyingFox** [comment]
Maybe you downloaded the wrong version or maybe it's just what your hardware support. You didn't list what you downloaded, your OS, or any details about your computer at all so it's hard to know.

**Reply by AverageMinecraftEnjoyer** [reply]
Oh, My bad I have an AMD Based System. I downloaded the onnxdirectML-cuda v.1.5.3.18a the one that is supposed to work on AMD , My specs are

R7 5700X
RX 6700XT
16GB 3200MHz 
OS: W11 Pro

I had an Nvidia GPU Before and they were available, but I downloaded the Cuda version of this app so I dismissed this.

**Reply by AverageMinecraftEnjoyer** [reply]
May I add that in discord sometimes I sound robotic and sometimes I sound perfectly fine, and when I use the voice changer in roblox its always perfectly fine (this is the only game I tested it in)

**Reply by AverageMinecraftEnjoyer** [reply]
I can show the settings I use too if you want
